masterlist.yaml

```diff
@@ -21851,3 +21851,10 @@ plugins:
     msg: [ *alreadyInOrFixedByBDSv3_0 ]
 
 ###### Anything new can go after this (If you're not sure where to put it) ######
+
+  - name: 'Omni-No-Useless-Papers-SSE.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/51468' ]
+    req: [ 'Unofficial Skyrim Special Edition Patch.esp' ]
```

Comment:
You don't need to add a module to requirements If it is listed in the file header Master Files.
LOOT reads the modules Master Files in the header and shows an error if any are missing.

![image](https://user-images.githubusercontent.com/1944639/131730139-d7ab1c09-a92b-4fa6-87af-7b4163444316.png)

```suggestion
```